components/cdmf-base-ui/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.types.listing/listing.js

```diff
@@ -43,30 +43,30 @@ function onRequest(context) {
                     deviceTypeLabel = configs[DTYPE_CONF_DEVICE_TYPE_KEY][DTYPE_CONF_DEVICE_TYPE_LABEL_KEY];
                 }
             }
-            deviceTypesList.push({
-                "hasCustTemplate": false,
-                "deviceTypeLabel": deviceTypeLabel,
-                "deviceTypeName": deviceTypes[i].name,
-                "deviceCategory": deviceCategory,
-                "deviceTypeId": deviceTypes[i].id
-            });
-            //if (deviceCategory == 'virtual'){
-            //    virtualDeviceTypesList.push({
-            //        "hasCustTemplate": false,
-            //        "deviceTypeLabel": deviceTypeLabel,
-            //        "deviceTypeName": deviceTypes[i].name,
-            //        "deviceCategory": deviceCategory,
-            //        "deviceTypeId": deviceTypes[i].id
-            //    });
-            //}else{
-            //    deviceTypesList.push({
-            //        "hasCustTemplate": false,
-            //        "deviceTypeLabel": deviceTypeLabel,
-            //        "deviceTypeName": deviceTypes[i].name,
-            //        "deviceCategory": deviceCategory,
-            //        "deviceTypeId": deviceTypes[i].id
-            //    });
-            //}
+            //deviceTypesList.push({
```

Comment:
Acknowledged and fixed with commit https://github.com/wso2/carbon-device-mgt/commit/0133cddc683fe7bf2c381a3e338d1c027f6ff1b0